numbers-quickstart/NumbersInCSharp/Program.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+//using System.Math;
+
+WorkWithIntegers();
+OrderPrecedence();
+TestLimits();
+WorkWithDoubles();
+WorkWithDecimals();
+AreaOfCircle();
+
+void WorkWithIntegers()
+{
+    Console.WriteLine("---WorkWithIntegers---");
+    int a = 18;
+    int b = 6;
+
+    // addition
+    int c = a + b;
+    Console.WriteLine(c);
+    // subtraction
+    c = a - b;
+    Console.WriteLine(c);
+
+    // multiplication
+    c = a * b;
+    Console.WriteLine(c);
+
+    // division
+    c = a / b;
+    Console.WriteLine(c);
+}
+
+void OrderPrecedence()
```

Comment:
All good, but in commercial code, we add /// code comments at the beginning of each public method and property. Try it in your code, put /// in front of each method and see what happens